Jellyfin.Server/Formatters/XmlOutputFormatter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Formatters;
+
+namespace Jellyfin.Server.Formatters
+{
+    /// <summary>
+    /// Xml output formatter.
+    /// </summary>
+    public class XmlOutputFormatter : TextOutputFormatter
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="XmlOutputFormatter"/> class.
+        /// </summary>
+        public XmlOutputFormatter()
+        {
+            SupportedMediaTypes.Add("text/xml");
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.net.mime.mediatypenames.text.xml?view=netcore-3.1 :(

Reply:
It's "text/xml;charset=UTF-8" that makes it work. It's defined in the emby.dlna

Reply:
so is `text/xml` not even needed?

Reply:
I'll double check in the morning, but i think it wasn't working because of the text/xml;charset=UTF8 - i'll double check in the morning.

Reply:
Can confirm - text/xml **NOT** needed only "text/xml;charset=UTF-8"